docs/diagrams/CalcBooleanCommand.puml

```diff
@@ -0,0 +1,22 @@
+@startuml
```

Comment:
This will need to be revised once @sevenseasofbri implements `CalculateBooleanCommand.execute()`. Consider revising the return of control of the sequence diagrams.

Reply:
Sure, will do.

Reply:
Already implemented - can be updated.